integrations/slack/events/app.go

```diff
@@ -45,3 +45,23 @@ func AppMentionHandler(l *zap.Logger, w http.ResponseWriter, body []byte, cb *Ca
 	}
 	return j.Event
 }
+
+// https://api.slack.com/events/app_home_opened
+// https://api.slack.com/surfaces/app-home
+func AppHomeOpenedHandler(l *zap.Logger, w http.ResponseWriter, body []byte, cb *Callback) any {
+	// Parse and return the inner event details.
+	var eventData map[string]any
+	if err := json.Unmarshal(body, &eventData); err != nil {
+		invalidEventError(l, w, body, err)
+		return nil
+	}
+
+	if event, ok := eventData["event"]; ok {
+		return event
+	}
+
+	l.Warn("Event key not found in the JSON for app_home_opened event",
+		zap.ByteString("json_body", body),
+	)
+	return nil
```

Comment:
```suggestion
	event, ok := eventData["event"]; ok
	if !ok {
		l.Warn("Event field not found in the Slack event app_home_opened", zap.ByteString("event", eventData))
		return nil
	}

	return event
```

Reply:
The ok after the semicolon is causing an error because !ok is checked immediately afterward. This can be simplified to avoid unnecessary double-checking.

Reply:
Another issue is zap.ByteString("event", eventData). The eventData should be marshaled, as zap.ByteString expects the raw JSON body. We can solve that by returning the original body parameter

Reply:
Oops, fixed both issues in the suggestion. Also remember to rename `event` to `innerEvent` and `eventData` to `event` before merging.

Reply:
Oops! Your fixes are ok, but I made a tiny suggestion on top of them.